pricing-page-exercise/onur/styles.css

```diff
@@ -0,0 +1,263 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-family: "Open Sans", sans-serif;
+    font-size: 16px;
+}
+body {
+    margin: 0;
+    padding: 0;
+}
+/* navbar section */
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1.25rem;
+    background: rgb(255, 255, 255);
+    height: 70px;
+    border-bottom: 1px solid rgb(222 226 230);
+    box-shadow: rgba(0, 0, 0, 0.05) 0px 7px 11px 0px;  
+}
+
+.heading-small {
+    font-size: 1.2rem;
+    font-weight: 400;
+    text-align: left;
+}
+
+.navbar ul {
+    display: flex;
+    align-items: center;
+    gap: 1.2rem; 
+    list-style-type: none;
+}
+
+.navlink {
+    text-decoration: none;
+    color: #363636;
+    font-size: 1rem;
+}
+.navlink:hover {
+    text-decoration:underline;
+    color:#363636;
```

Comment:
Try not to mix different color units in the same project. It's the most common to use HEX (can be combined with rgba if opacity is needed) or rbg&rbga.